pkg/eval/external_cmd_test.go

```diff
@@ -0,0 +1,41 @@
+package eval
+
+import (
+	"os"
+	"testing"
+)
+
+func TestBuiltinFnExternal(t *testing.T) {
+	tmpHome, cleanup := InTempHome()
+	defer cleanup()
+
+	mustMkdirAll("dir")
+	mustCreateEmpty("cmdx")
+	mustWriteFile("cmd1", []byte("#!/bin/sh"), 0755)
+	mustWriteFile("cmd2", []byte("#!/bin/sh"), 0755)
+	mustWriteFile("cmd3", []byte(""), 0755)
+	mustCreateEmpty("file")
+
+	os.Setenv("PATH", tmpHome+":"+os.Getenv("PATH"))
+	Test(t,
+		That(`resolve external`).Puts("$external~"),
```

Comment:
This test case doesn't belong here.

Reply:
Agreed. Moved to *pkg/eval/builtin_fn_misc_test.go*.